### PR TITLE
Datadog - Add device as a tag if it's present as a field in the series object

### PR DESCRIPTION
**Pull request by PerGon**
For some reason that I don't understand, the `datadog-agent` (at least v7) removes the `device` tag from the `Tags` field and adds it as it's own field in the Series object - as far as I can tell, this is not mentioned in the [datadog documentation to submit metrics (check v1)](https://docs.datadoghq.com/api/latest/metrics/#submit-metrics).

In the `datadog-agent` code, you can see the `device` tag being removed from the `Tags` field [in here](https://github.com/DataDog/datadog-agent/blob/0ada7a97fed6727838a6f4d9c87123d2aafde735/pkg/metrics/series.go#L84-L105).

What this PR does is to add what is in the device field value as a label.

### CODE REVIEW

File: app/vmagent/datadog/request_handler.go [hunk 1]
```diff
@@ -56,6 +56,12 @@ func insertRows(at *auth.Token, series []parser.Series, extraLabels []prompbmars
 			Name:  "host",
 			Value: ss.Host,
 		})
+		if len(ss.Device) > 0 {
```
Comment (dmitryk-dk) [inline app/vmagent/datadog/request_handler.go:59]: I think we can simply check 
```if ss.Device != ""```
The same in other places

**Review by dmitryk-dk — COMMENTED** [review]
Looks good! Added simple comment

**Review by valyala — APPROVED** [review]

**Comment by valyala** [comment]
@PerGon , thanks for the pull request!

**Comment by valyala** [comment]
FYI, this pull request has been included in [VictoriaMetrics v1.85.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases)

**Pull request merged**